initial_setup.py

```
@@ -0,0 +1,667 @@
+        username = Prompt.ask("[bold]Admin username[/bold]", default="admin")
+
+        # Check if user exists
+        if os.path.exists(database_path):
+            conn = sqlite3.connect(database_path)
+            cursor = conn.cursor()
+            try:
+                cursor.execute(
+                    "SELECT COUNT(*) FROM users WHERE username = ?", (username,)
+                )
+                user_exists = cursor.fetchone()[0] > 0
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)